### PR TITLE
book review part2

### PR DESCRIPTION
**Pull request by hexcoder-**
The rest of my document review. Please review to make sure that I have not changed the semantics unintentionally, thanks!

### CODE REVIEW

File: docs/src/advanced_features/concolic.md [hunk 1]
```diff
@@ -92,18 +92,18 @@ In hybrid fuzzing, we combine this tracing + solving approach with more traditio
...
-These callbacks allow the runtime to construct a trace that similar to the previous example.
+These callbacks allow the runtime to construct a trace that is similar to the previous example.
 
 ### SymCC and its Runtimes
 
 SymCC ships with 2 runtimes: 
 
- * a 'simple' runtime that attempts to solve any branches it comes across using [Z3](https://github.com/Z3Prover/z3/wiki) and
- * a [QSym](https://github.com/sslab-gatech/qsym)-based runtime, which does a bit more filtering on the expressions and also solves using Z3.
+ * a 'simple' runtime that attempts to negate any branch conditions it comes across using [Z3](https://github.com/Z3Prover/z3/wiki) and
```
Comment (domenukk) [inline docs/src/advanced_features/concolic.md:101]: Isn't solving the correct term here?

File: docs/src/advanced_features/nyx.md [hunk 2]
```diff
@@ -90,7 +90,7 @@ let mut helper = NyxHelper::new(
     cpu_id,    // current cpu id
     true,      // open snap_mode
     parallel_mode, // open parallel mode
-    Some(parent_cpu_id.id as u32), // the cpu-id of master instance, there is only one master instance, other instances will be treated as slaved
+    Some(parent_cpu_id.id as u32), // the cpu-id of master instance, there is only one master instance, other instances will be treated as slaves
```
Comment (domenukk) [inline docs/src/advanced_features/nyx.md:93]: We should use `main` and `secondary` instance here, to match afl++

**Review by domenukk — APPROVED** [review]

File: docs/src/advanced_features/concolic.md (hunk 1 above)
Comment (domenukk) [reply docs/src/advanced_features/concolic.md]: Using solved for now, hope I'm understanding this correctly.

**Comment by domenukk** [comment]
Thank you! :)

File: docs/src/advanced_features/concolic.md (hunk 1 above)
Comment (hexcoder-) [reply docs/src/advanced_features/concolic.md]: solving is fine when it is explicitly stated what is being solved IMO. In this case the negation of the branch conditions encountered so far in order to reach yet unreached branches with the next input data.

Comment (domenukk) [reply docs/src/advanced_features/concolic.md]: Proposed a wording in #986

**Pull request merged**